### PR TITLE
Fixed test ENV variable, CREATE module tests created

### PR DESCRIPTION
**Pull request by dhruv2000**
fixed environment variable

### CODE REVIEW

**Review by HamishPierpont — APPROVED** [review]

**Review by HamishPierpont — DISMISSED** [review]
LGTM

File: mocks/module.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//eslint-disable-next-line
+const createSampleModule = () => {
+  return {
+    name: 'Stacks/Lists/Queues',
+    number: 1.1
```
Comment (HamishPierpont) [inline mocks/module.ts:5]: Should we force these numbers to be integers? I'm concerned about roundoff error

Comment (dhruv2000) [reply mocks/module.ts:5]: Yeah that was definitely something I wanted to discuss with you guys

**Review by marcgabe15 — APPROVED** [review]

**Review by HamishPierpont — APPROVED** [review]
LGTM

**Pull request merged**